### PR TITLE
Some additions and enhancements to the local installation.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: 06-getting-your-class-going-with-jupyter.md [hunk 1]
```diff
@@ -9,52 +9,52 @@ internet.
...
-for everyone, and don't need to worry that not everyone can get it to
-work on their own.
-
+Two other easily installable software packages that can run Jupyter notebooks
+are [nteract](https://nteract.io/) and [Hydrogen](https://nteract.io/atom).
+nteract is installed by downloading a binary installer from their website and
+double-clicking the installation file. nteract's simple user interface make it
+an excellent choice for students new to computer programming. Once nteract is
+installed, any Jupyter notebook on a student's local system can be
+double-clicked and it will open within nteract. Anaconda does not default to
```
Comment (zingale) [inline 06-getting-your-class-going-with-jupyter.md]: I don't think it is a double-click for every OS

Comment (moorepants) [reply 06-getting-your-class-going-with-jupyter.md]: Double comments...

Comment (willingc) [reply 06-getting-your-class-going-with-jupyter.md]: A bit confused on the Anaconda sentence. nteract does allow double clicking on Mac, Windows, Linux to launch a notebook within nteract. Is the sentence about Anaconda saying its confusing that Anaconda does not allow double click to launch Jupyter? I would recommend striking the Anaconda sentence within the nteract section.

**Comment by dsblank** [comment]
@moorepants So it sounds like there is consensus on your changes; but now it needs a rebase. Do you want to resolve and merge?

**Comment by willingc** [comment]
I've fixed the merge conflicts. Merging. Thanks a bunch @moorepants 🚲

**Comment by willingc** [comment]
Oops sorry for the accidental close/reopen.

**Comment by moorepants** [comment]
Sorry, I was not connected for the last couple of days. Thanks for handling it.

**Pull request merged**